### PR TITLE
Homepage restyle

### PR DESCRIPTION
**Pull request by tnylea**
Hey Guys,

I've made some changes to the styles of the Codeception homepage.

You can take a look at the changes from the screenshots below:

Homepage redesign screenshot
Homepage redesign mobile screenshott

I've updated a new google font, create a new homepage hero image, made sure it's mobile compatible, and fixed the correct date 2011-2015 in the footer of the site.

Let me know if you have any questions :)

Thanks :+1:

### CODE REVIEW

**Comment by tnylea** [comment]
Hey @DavertMik, I've submitted a new design for the codeception webpage. I'll have many more to go. Just wanted to see if you liked the design and could do the merge. Let me know if you think I should change anything else :)

Really appreciate it.

**Comment by DavertMik** [comment]
Hi, thank you! Awesome job! Looks really elegant. 
I will take a detailed look on it and probably merge it on Monday.

File: _layouts/bootstrap.html [hunk 1]
```diff
@@ -173,7 +173,7 @@
     </div>
 </div>
 <div id="copyright">
-    © 2011&ndash;<span data-toggle="tooltip" alt="" title="YOU BROKE MY PHP! FIX IT ASAP!">&lt;?= date('Y') ?&gt;</span>
+    © 2011&ndash;{{ site.time | date: '%Y' }}
```
Comment (javigomez) [inline _layouts/bootstrap.html]: he he he, @DavertMik  nobody understands your Easter egg https://github.com/Codeception/codeception.github.com/pull/30

Comment (DavertMik) [inline _layouts/bootstrap.html]: yeah :(

**Comment by DavertMik** [comment]
Thanks, I did a few changes: I reverted font back to Philosopher as I got used to it, and updated hero pic with photos of my friend photographer. I hope you didn't mind ) Thanks, site looks much better now

**Comment by DavertMik** [comment]
Btw, @tnylea  can I get your twitter to mention you?

**Pull request merged**